src/types/components/ComponentsTypes.ts

```diff
@@ -8,11 +8,13 @@ export interface ISignUpForm {
 	email: string;
 	password?: string;
 	re_password?: string;
+	agreementField?: boolean;
 }
 
 export interface ISignInForm {
 	email: string;
 	password?: string;
+	isAuth?: boolean;
 }
 
```

Comment:
export interface ICorrectSignInForm {
	email: string;
	password: string;
}

this type needs to be added